package.json

```diff
@@ -39,21 +39,20 @@
     "compressible": "^2.0.9",
     "compression": "^1.6.2",
     "electron": "^1.6.1",
+    "eslint-plugin-html": "^2.0.3",
```

Comment:
This plugin is for scanning JavaScript within the script tags of HTML files.

In Cesium's case this is needed to scan `'Apps/Sandcastle/gallery/*.html'`